src/NuGet.Core/NuGet.Protocol/Resources/RawSearchResourceV3.cs

```diff
@@ -113,12 +116,175 @@ public virtual async Task<JObject> SearchPage(string searchTerm, SearchFilter fi
             throw new FatalProtocolException(Strings.Protocol_MissingSearchService);
         }
 
+        [Obsolete]
```

Comment:
```suggestion
        [Obsolete("Use PackageSearchResource instead (via SourceRepository.GetResourceAsync<PackageSearchResource>")]
```

It's really not obvious to anyone using this API what they should be using instead.

Reply:
Ok. Done.